server/sonar-server/src/main/java/org/sonar/server/rule/ws/RuleWsSupport.java

```diff
@@ -44,7 +44,7 @@ public RuleWsSupport(DbClient dbClient, UserSession userSession, DefaultOrganiza
     this.defaultOrganizationProvider = defaultOrganizationProvider;
   }
 
-  public void checkQProfileAdminPermission() {
+  public void checkQProfileAdminPermissionOnDefaultOrganization() {
```

Comment:
The change seemed too small to me to deserve a separate commit, however I'm going to keep this consideration in mind for the next changes I do.